Frontity commands not working without `typescript`

**_Bug report_**

- [x] I've searched for existing issues
- [x] I've checked documentation: https://docs.frontity.org
- [x] I've checked community forum: https://community.frontity.org

After the new release, if you haven't `typescript` install, frontity commands like `create` or `dev` don't work.

**Describe the expected behavior**

They should work as before even if typescript is not installed.

**Describe the observed behavior**

Right now it doesn't work and it returns this error:
```
Cannot find module 'typescript'
Require stack:
- /Users/mario/.npm/_npx/3939/lib/node_modules/frontity/node_modules/ts-node/dist/index.js
- /Users/mario/.npm/_npx/3939/lib/node_modules/frontity/dist/src/cli/index.js
```

If I install `typescript`, it works properly.

--- a/.changeset/young-cobras-serve.md
+++ b/.changeset/young-cobras-serve.md
@@ -1,0 +1,5 @@
+---
+"frontity": patch
+---
+
+Fix missing typescript dependency in the frontity package. This package is the one responsible to inject typescript in a Frontity Project and the Frontity CLI.

--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
     "lint-staged": "^8.1.4",
     "prettier": "^1.16.4",
     "ts-jest": "^24.0.2",
-    "typescript": "^3.7.5"
+    "typescript": "^3.8.2"
   },
   "prettier": {}
 }

--- a/packages/frontity/package.json
+++ b/packages/frontity/package.json
@@ -57,7 +57,8 @@
     "simple-entity-decode": "^0.0.3",
     "tar": "^4.4.8",
     "ts-node": "^8.0.3",
-    "typed-emitter": "^0.2.0"
+    "typed-emitter": "^0.2.0",
+    "typescript": "^3.8.2"
   },
   "devDependencies": {
     "@frontity/core": "^1.5.0",

